cypress/integration/manageUsers.spec.js

```diff
@@ -0,0 +1,90 @@
+// <reference types="Cypress" />
+import { USERS } from '../support/selectorsSettings';
+
+describe('Filtering', () => {
+  beforeEach(() => {
+    localStorage.setItem('accessToken', 'TEST123');
+  });
+
+  it('Should open the manage users screen', () => {
+    cy.server();
+    cy.getManageSignalsRoutes();
+
+    cy.visitFetch('/manage/incidents/');
+
+    cy.wait('@getFilters');
+    cy.wait('@getCategories');
+    cy.wait('@getSignals');
+    cy.wait('@getUserInfo');
+
+    cy.openMenu();
+    cy.contains('Instellingen').click();
+    cy.contains('Gebruikers').click();
+    cy.get('[aria-label="Menu"]').click();
+
+    cy.url().should('include', '/instellingen/gebruikers/page/1');
+  });
+
+  // This test case can be executed when we start every test with a clean database
+  it.skip('Should add a user', () => {  
+    cy.get(USERS.inputMail).type('sia@fakemail.nl');
+    cy.get(USERS.inputVoornaam).type('Simon');
+    cy.get(USERS.inputAchternaam).type('Ia');
+    cy.contains('div', 'GGD').find('input').check();
+    cy.get(USERS.inputNotitie).type('Dit is de belangrijkste gebruiker van SIA, let op!');
+    cy.get(USERS.buttonOpslaan).click();
+  });
+
+  it('Should add a department to a user', () => { 
+    cy.server();
+    cy.route('/signals/v1/private/users/*').as('getUser');
+    cy.route('PATCH', '/signals/v1/private/users/*').as('patchUser');
+    cy.get(USERS.userRow).eq(1).click();
+    cy.wait('@getUser');
+    cy.url().should('include', '/instellingen/gebruiker/');
```

Comment:
The URL should also contain an id. Can `.should('include', '/instellingen/gebruiker/')` also contain a regex? I think `match` can do that:
```jsx
cy.url().should('match', /\/instellingen\/gebruiker\/\d+^/');
```